internal/engine/build_and_deployer_test.go

```diff
@@ -466,7 +466,7 @@ func TestIgnoredFiles(t *testing.T) {
 	})
 }
 
-func TestCustomBuild(t *testing.T) {
+func TestCustomBuildWithFastBuild(t *testing.T) {
```

Comment:
do we want a non-fast-build test?

(definitely not a blocker for this PR since this PR isn't causing that problem, just revealing it)

Reply:
Yeah I added to my todo list to add more tests here now that I have a better understanding of how it works.